### PR TITLE
Add basic support for arrays of primitives

### PR DESCRIPTION
**Pull request by tgoyne**
All of the array-related functionality I could find navigating through the app now works to at least some degree, although it's distinctly unpolished. Probably needs a new RC release so that it doesn't have to point at a branch of realm-cocoa.

### CODE REVIEW

File: RealmBrowser/Support/RLMModelExporter.m [hunk 1]
```diff
@@ -427,6 +467,24 @@ + (NSString *)swiftDefinitionForProperty:(RLMProperty *)property
...
+            case RLMPropertyTypeBool:   type = @"Bool"; break;
+            case RLMPropertyTypeInt:    type = @"Int"; break;
+            case RLMPropertyTypeFloat:  type = @"Float"; break;
+            case RLMPropertyTypeDouble: type = @"Double"; break;
+            case RLMPropertyTypeString: type = @"String"; break;
+            case RLMPropertyTypeData:   type = @"NSData"; break;
+            case RLMPropertyTypeDate:   type = @"NSDate"; break;
+            case RLMPropertyTypeObject: type = property.objectClassName;
+            case RLMPropertyTypeAny: return @"/* Error! 'Any' properties are unsupported in Swift. */";
+            case RLMPropertyTypeLinkingObjects: return @"";
```
Comment (bdash) [inline RealmBrowser/Support/RLMModelExporter.m]: Should this not return something other than an empty string?

Comment (tgoyne) [reply RealmBrowser/Support/RLMModelExporter.m]: This code will never actually encounter a LinkingObjects property for a whole bunch of reason and the existing cases below just return an empty string.

Comment (bdash) [reply RealmBrowser/Support/RLMModelExporter.m]: Is that true of the equivalent Objective-C code path as well? It currently returns something other than an empty string, which is why I asked about this one.

Comment (tgoyne) [reply RealmBrowser/Support/RLMModelExporter.m]: Yes. There's no way to get computed properties into the schema that the browser works with, so all the code for doing things with RLMPropertyTypeLinkingObjects properties is dead.

File: Podfile [hunk 2]
```diff
@@ -5,8 +5,8 @@ use_frameworks!
 
 target 'RealmBrowser' do
     pod 'AppSandboxFileAccess'
-    pod 'Realm', '~> 3.0.0-beta'
-    pod 'RealmConverter'
+    pod 'Realm', '~> 3.0.0-rc.1'
+    pod 'RealmConverter', git: 'https://github.com/realm/realm-cocoa-converter', branch: 'mar/cocoa-3.0.0-beta.4'
```
Comment (bdash) [inline Podfile]: Are you ok with us figuring out how to release RealmConverter at a later date?

**Pull request merged**